interface/wx/dirctrl.h

```diff
@@ -174,6 +187,14 @@ class wxGenericDirCtrl : public wxControl
     */
     virtual wxDirFilterListCtrl* GetFilterListCtrl() const;
 
+    /**
+        Returns a pointer to the wxMenu which was popped up when the user right-clicked on an item.
+        This is useful for adding a separator to the menu.
```

Comment:
I guess this means "because other items should be added with NewMenuItem()", but it'd be better to say so explicitly.

If we keep this API, of course.